### PR TITLE
test(clusterconfig): add unit test for ingresses.go and rename it to gather_cluster_ingress.go 

### PR DESCRIPTION
**Pull request by rluders**
This PR adds a simple unit test for the `ingress` gatherer and renames its file from `ingresses.go -> gather_cluster_ingress.go`

## Categories

- [ ] Bugfix
- [ ] Data Enhancement
- [ ] Feature
- [ ] Backporting
- [X] Others (CI, Infrastructure, Documentation)

## Sample Archive

None

## Documentation

None

## Unit Tests

- `pkg/gatherers/clusterconfig/gather_cluster_ingress_test.go`

## Privacy

Yes. There are no sensitive data in the newly collected information.

## Breaking Changes

No

## References

https://issues.redhat.com/browse/CCXDEV-10106

### CODE REVIEW

File: pkg/gatherers/clusterconfig/gather_cluster_ingress_test.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_gatherClusterIngress(t *testing.T) {
+	tests := []struct {
+		name         string
+		config       *configv1.Ingress
```
Comment (tremes) [inline pkg/gatherers/clusterconfig/gather_cluster_ingress_test.go:18]: Calling the attribute `config` doesn't make much sense IMHO. `ingress` or `ingressDefinition` would be better IMHO.

File: pkg/gatherers/clusterconfig/gather_cluster_ingress_test.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+					Item: record.ResourceMarshaller{
+						Resource: &configv1.Ingress{ObjectMeta: metav1.ObjectMeta{Name: "cluster"}},
+					},
+				},
+			},
+			wantErrCount: 0,
+		},
+		{
+			name:         "cluster ingress not found",
+			config:       &configv1.Ingress{},
```
Comment (tremes) [inline pkg/gatherers/clusterconfig/gather_cluster_ingress_test.go:37]: Minor improvement suggestion. We could provide some name here to make more obvious that only "cluster" Ingress is gathered, but it's just a detail.

**Comment by tremes** [comment]
Simple unit test. Reviewed.
/lgtm

**Comment by rluders** [comment]
This is just a unit test + file renaming should not affect operator functionality. No need for e2e.
/override ci/prow/e2e-agnostic-upgrade ci/prow/insights-operator-e2e-tests

**Comment by tremes** [comment]
This is just unit test update.
/override ci/prow/e2e-agnostic-upgrade

**Pull request merged**